apps/ui/src/helpers/utils.ts

```diff
@@ -27,18 +27,6 @@ import IHGlobeAlt from '~icons/heroicons-outline/globe-alt';
 
 const IPFS_GATEWAY: string =
   import.meta.env.VITE_IPFS_GATEWAY || 'https://cloudflare-ipfs.com';
-const ADDABLE_NETWORKS = {
-  59140: {
-    chainName: 'Linea Goerli test network',
-    nativeCurrency: {
-      name: 'LineaETH',
-      symbol: 'ETH',
-      decimals: 18
-    },
-    rpcUrls: ['https://rpc.goerli.linea.build'],
-    blockExplorerUrls: ['https://goerli.lineascan.build']
-  }
-};
```

Comment:
Maybe we could keep this commented out so we keep the logic for adding new networks, might be useful in the future.